src-backend/registration/tests/test_signup.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+import json
+
+
+class SignupTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_programmatically_created_user_has_token(self):
```

Comment:
This should be separated into a separate test class. It is a unit test vs. functional test.